chemprop/features/utils.py

```diff
@@ -53,3 +54,21 @@ def load_features(path: str) -> np.ndarray:
         raise ValueError(f'Features path extension {extension} not supported.')
 
     return features
+
+
+def load_atom_features(path: str) -> List[np.ndarray]:
+    """
+    Loads features saved in a .pkl file
+    :param path: Path to file containing atomwise features
+    :return: A list of 2D array
+    """
```

Comment:
Just for consistency.

```suggestion
    """
    Loads features saved in a .pkl file.

    :param path: Path to file containing atomwise features.
    :return: A list of 2D array.
    """
```